admin-ui/app/project/index.tsx

```diff
@@ -363,8 +301,7 @@ const ProjectHomeComponent = () => {
         itemsCallBackHandler={itemsCallBackHandler}
         filterSelected={filterSelected}
         setFilterSelected={setFilterSelected}
-        chip={chip}
-        setChip={setChip}
+        setFilterChipType={setFilterChipType}
```

Comment:
what is set type, it could be handler, and handle can check the valid type and update the state